frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -459,6 +459,10 @@ machinery-MicrosoftTerminology--visit-microsoft = MICROSOFT
         Visit Microsoft Terminology Service API.
         © 2018 Microsoft Corporation. All rights reserved.
 
+## Machinery Microsoft Terminology
+## Shows the translation source from Microsoft Terminology.
+terms-MicrosoftTerminology--visit-microsoft = MICROSOFT
+    .title = Copy Into Translation
```

Comment:
You can remove the comment, because it's pasted from elsewhere :) and move the string further down the file, just above `term-Term--for-example`.

You'll also need to update the id to match the one used in the component (`term-Term--copy`) and remove the message value (`MICROSOFT`), because it's not used.